tests/Diagnostics.Tests.Integration.RuntimeHost/TestData.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ListDetector": [
```

Comment:
possible to checkin the empty file and not have actual resources (with subid etc) here ?

Reply:
have the real values during test runtime